crypto/fipsmodule/PQREADME.md

```diff
@@ -0,0 +1,65 @@
+
+| Algorithm        | Public Key (B) | Private Key (B) |Ciphertext(B) |
+|--------------------|-------------------|---------------------|-------------------|
+| ML-KEM-512   |                   800|                   1632|                   768|
+| ML-KEM-768   |                 1184|                   2400|                 1088|
+| ML-KEM-1024 |                 1568|                   3168|                 1568|
+
+These three parameter sets were designed to meet security strength categories defined by NIST. These security strength categories are explained further in SP 800-57, Part 1. Concretely, ML-KEM-512 is claimed to be in security category 1, ML-KEM-768 is claimed to be in security category 3, and ML-KEM-1024 is claimed to be in security category 5.
+
+Performance benchmarks for key generation, encapsulation, and decapsulation are included for ML-KEM within the `speed` tool. To run:
+
+```aws-lc-build % ./tool/bssl speed -filter ML-KEM```
+
+#### KyberR3
+
+Round 3 Kyber (KyberR3) was added to AWS-LC in September 2021 ([README](https://github.com/aws/aws-lc/blob/main/crypto/kyber/README.md)). Once all existing deployments of Kyber are migrated over to ML-KEM we will be removing support for Kyber from AWS-LC.
+
+### FIPS 204: Module-Lattice-Based Digital Signature Algorithm (ML-DSA)
+
+| Algorithm        | Public Key (B) | Private Key (B) |Signature (B) |
+|--------------------|-------------------|---------------------|------------------|
+| ML-DSA-44      |                 2560|                   1312|                2420|
+| ML-DSA-65      |                 4032|                   1952|                3309|
+| ML-DSA-87      |                 4896|                   2592|                4627|
+
+The parameter set ML-DSA-44 is claimed to be in security strength category 2, ML-DSA-65 is claimed to be in category 3, and ML-DSA-87 is claimed to be in category 5.
+
+## AWS-LC Post-Quantum Integrations
+
+### Hybrid Post-Quantum TLS Specifications
+
+To utilize Post-Quantum key exchange in TLS we recommend using our open-source TLS implementation s2n-tls that now supports Hybrid key exchange in TLS 1.3 (draft-ietf-tls-hybrid-design). s2n-TLS also provides support for Post-Quantum hybrid ECDHE-MLKEM Key Agreement for TLSv1.3 (draft-kwiatkowski-tls-ecdhe-mlkem) with a proposal for new key share identifies for x25519 and ML-KEM-768.
+
+
+| Supported Group             | IANA ID (Hex) | IANA ID (Dec) |
+|---------------------------------|--------------------|--------------------|
+| x25519_kyber512            |              0x2f39|                12089|
+| p256_kyber512                |              0x2f3a|                12090|
+| X25519Kyber768Draft00|             0x6399|                25497|
+| X25519Kyber768Draft00|             0x639a|                25498|
+|SecP256r1MLKEM768      |             0x11eb|                  4587|
+|X25519MLKEM768           |              0x11ec|                  4588|
+
+
+## AWS Java V2 SDK
```

Comment:
I think this needs to be `###` to match the header level for integrations.

Reply:
good spot, fixed in [6ee84ff](https://github.com/aws/aws-lc/pull/1932/commits/6ee84ffe30fa5c3a959b12e7f6feba51925507c0)